addons/snackpacks/CfgWeapons.inc

```diff
@@ -0,0 +1,118 @@
+#define CLASSES_WEAPONS         \
+    "SWST_GummiShark"         \
+    , "SWST_TigerThymes"  \
+    , "SWST_SonicBooms"
+
+class CfgWeapons {
+    class CBA_MiscItem_ItemInfo;
+    class ACE_Can_Spirit;
+    class ACE_Sunflower_Seeds;
+    class SWST_Base_Drink : ACE_Can_Spirit {
+        ITEM_META(1);
+        picture = QPATHTOF(data\ui\drink_ui_generic.paa);
+        displayName = "Base Drink Item";
+        descriptionShort = "Drinks are like snacks.<br/>Not that we play with real rations on anyway?";
+        acex_field_rations_consumeText = "Getting sippy of %1...";
+        acex_field_rations_consumeTime = 5;
+        acex_field_rations_thirstQuenched = 5;
+        class ItemInfo: CBA_MiscItem_ItemInfo {
+            mass = 1;
+        };
+    };
+    class SWST_Base_Snack : ACE_Sunflower_Seeds {
+        ITEM_META(1);
+
+        acex_field_rations_consumeText = "Monching %1...";
+        acex_field_rations_consumeTime = 5;
+        acex_field_rations_hungerSatiated = 2;
+
+        displayName = "Base Snack Item";
+        descriptionShort = "Snacks are not meals.<br/>They weigh very little and are just for funsies.";
+        model = QPATHTOF(data\snack_generic.p3d);
+
+        picture = QPATHTOF(data\ui\snack_ui_generic.paa);
+        hiddenSelections[] = {"camo"};
+		hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_generic_co.paa)};
+
+        class ItemInfo: CBA_MiscItem_ItemInfo {
+            mass = 1;
+        };
+    };
+    class SWST_GummiShark : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Gummi Shorks";
+        descriptionShort = "A pouch of Loreli-brand gummy sharks. They're sweet with a little bite of sour.";
+
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_shorks_co.paa)};
+        picture = QPATHTOF(data\ui\snack_ui_shorks.paa);
+    };
+    class SWST_TigerThymes : SWST_Base_Snack {
+        ITEM_META(2);
+
+        displayName = "Tiger Thymes";
+        descriptionShort = "A pouch of savory jerky snacks. Herbacious flavor.<br/>May cause cheerfulness even in the face of hopeless odds.";
+
+        hiddenSelections[] = {"camo"};
```

Comment:
Remove all the `hiddenSelections[] = {"camo"}` except for the ones in the bases, since they aren't needed.